server/utils/users/utils.ts

```diff
@@ -0,0 +1,51 @@
+import type { SelectOption } from '@approved-premises/ui'
```

Comment:
The other utils for this view are in the `utils/users/userManagement.ts` file so I think it would make sense for this function to be in that file too